### PR TITLE
Expose Fastlane's simulator reset actions

### PR DESCRIPTION
**Pull request by danramteke**
Useful in conjunction with running tests with `scan`!

### CODE REVIEW

File: fastlane/lib/fastlane/actions/reset_simulators.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        FastlaneCore::Simulator.reset_all
+        UI.success('Simulators reset')
+      end
+
+      def self.description
+        "Shutdown and reset running simulators"
+      end
+
+      def self.available_options
+        nil
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/reset_simulators.rb]: Could you change this to be `[]`

**Comment by KrauseFx** [comment]
This is perfect, thank you. I added one comment before it's ready to be merged. Also, could you squash the commits to one?

**Comment by danramteke** [comment]
Great! `available_options` returns `[]` instead of `nil` and I squashed all the commit!

**Comment by danramteke** [comment]
@KrauseFx hrm something went amiss with the Dangerfile - I don't think that's related to my code...

**Comment by KrauseFx** [comment]
Great, thanks for updating the PR :+1: Don't worry about danger here :rocket:

**Pull request merged**